official/transformer/model/embedding_layer.py

```diff
@@ -53,19 +68,25 @@ def call(self, x):
         locations of the padding tokens in x.
     """
     with tf.name_scope("embedding"):
-      embeddings = tf.gather(self.shared_weights, x)
+      # Create binary mask of size [batch_size, length]
+      mask = tf.to_float(tf.not_equal(x, 0))
+
+      if self.method == "gather" and False:
```

Comment:
Might want to remove the `and False` here

Reply:
...oops.